src/main/java/org/alliancegenome/curation_api/interfaces/crud/GeneCrudInterface.java

```diff
@@ -21,12 +34,47 @@
 public interface GeneCrudInterface extends BaseCurieCrudInterface<Gene>, BaseDTOCrudControllerInterface<Gene, GeneDTO> {
 
 	@Override
-	@JsonView(View.FieldsAndLists.class)
+	@GET
+	@Path("/{curie}")
+	@JsonView(View.GeneView.class)
 	public ObjectResponse<Gene> get(@PathParam("curie") String curie);
 
 	@POST
 	@Path("/bulk/genes")
 	@JsonView(View.FieldsAndLists.class)
 	public APIResponse updateGenes(List<GeneDTO> geneData);
+		
+	@Override
+	@PUT
+	@Path("/")
+	@JsonView(View.GeneView.class)
+	public ObjectResponse<Gene> update(Gene entity);
+	
+	@Override
+	@POST
+	@Path("/")
+	@JsonView(View.GeneView.class)
+	public ObjectResponse<Gene> create(Gene entity);
+	
+	@Override
+	@POST
+	@Path("/find")
+	@Tag(name = "Database Search Endpoints")
+	@JsonView(View.GeneView.class)
+	public SearchResponse<Gene> find(
+			@DefaultValue("0") @QueryParam("page") Integer page,
+			@DefaultValue("10") @QueryParam("limit") Integer limit,
+			@RequestBody HashMap<String, Object> params);
+	
+	@Override
+	@POST
+	@Path("/search")
+	@Tag(name = "Elastic Search Endpoints")
+	@JsonView({View.GeneView.class})
```

Comment:
Could probably promote `@JsonView({View.GeneView.class})` so the class level.

Reply:
I think we tried that with the AlleleView.class and couldn't get it to work.  There must be a way of doing that to avoid all these overrides though - maybe something I can look at in the off release sprint.